tests/acceptance/generate-component.spec.js

```diff
@@ -133,7 +133,7 @@ describe('Acceptance: ng generate component', function () {
     });
 
   it('ng generate component ..' + path.sep + 'my-comp from root dir will fail', () => {
-    return ng(['generate', 'component', '..' + path.sep + 'my-comp']).then(() => {
+    return ng(['generate', 'component', '..' + path.sep + 'my-comp']).catch(() => {
```

Comment:
Setting `null` as the success handler will report false positives. 

For instance, imagine `ng` command *doesn't* fail. The promise will be unrejected and thus the test passes whereas it should fail.

You have to replace the `null` with a promise rejection, like `() => new SilentError()` or `() => new Promise.reject()`.